packages/db-ui-elements-stencil/src/components/db-input/db-input.tsx

```diff
@@ -178,6 +180,11 @@ export class DbInput {
           aria-labelledby={this.input_id + '-label'}
           data-variant={this.variant}
           onChange={(event) => this.handleChange(event)}
+          onInput={(event) => {
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Why do we need a `@ts-ignore here`?

Reply:
It's always good to ignore TS

Reply:
I don't know why that ignore has been added in the first place. I've removed it and now it still looks fine.